.vitepress/errors.data.ts

```diff
@@ -0,0 +1,8 @@
+import { defineLoader } from 'vitepress'
+import { errorMessages } from '@vue/compiler-core'
```

Comment:
We should be including runtime errors here as well. In fact, runtime errors are more important than compiler errors because most of the time compilers run in dev mode.

Reply:
Runtime errors didn't be exported. We should do it first.

Reply:
`ErrorTypeStrings` is not exported in `runtime-core`, but exporting will increase the bundle size of vue itself at the same time. Because vue re-exported all of `runtime-core`.

Can we have a better way of exporting? How about exporting errors in another package entry, like `@vue/runtime-core/errors`

Reply:
Added runtime errors.